src/openrct2/interface/Screenshot.cpp

```diff
@@ -86,6 +86,20 @@ static sint32 screenshot_get_next_path(char *path, size_t size)
     rct2_time currentTime;
     platform_get_time_local(&currentTime);
 
+#ifdef _WIN32
```

Comment:
We use four spaces, not tabs.

Reply:
Man, so do I. I need to update Visual Studio.

Reply:
I'll fix everything. Do not push to your branch.

Reply:
Gah, I wish I had gotten that message 30 seconds ago

Reply:
I'm going to go read some Github tutorials because I pretty clearly don't know what I'm doing.

Reply:
Well, don't look up what I did with Git when I just started here...

Reply:
In any case: in this project, you rebase your branch off our develop when it cannot be merged as-is. You never merge a branch into yours.

Reply:
Some other stuff I changed:
- In C++, we use `nullptr`, not `NULL`.
- We do `x == 7`, not `7 == x`.

Reply:
Ok! So locally I'd do a `git rebase OpenRCT2\develop`?

Github has a really attractive "this doesn't merge, fix it?" button that seemed the obvious route.

Reply:
Ok, I'm done. Could you test it real quick? I don't have Windows.